EncoreTickets.SDK/Api/Results/Query.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace EncoreTickets.SDK.Api.Results
+{
+    public class Query
```

Comment:
Not a very helpful class for now. Is it expected to be contentless?

Reply:
This class corresponds to a part of a response that is returned from the API, and yes, now it is empty.